src/effects/Effect.cpp

```diff
@@ -1340,7 +1334,7 @@ bool Effect::ProcessTrack(EffectSettings &settings,
    decltype(mBufferSize) outputBufferCnt = 0;
    bool cleared = false;
 
-   auto chans = std::min<unsigned>(mNumAudioOut, mNumChannels);
+   auto chans = std::min<unsigned>(GetAudioOutCount(), mNumChannels);
```

Comment:
Number of output channels could be passed as a function argument to avoid possible diverges